dockerfiles/fedora-mock.Dockerfile

```diff
@@ -24,7 +24,7 @@ RUN useradd -m user
 RUN usermod -aG wheel user && echo '%wheel ALL=(ALL) NOPASSWD: ALL' > /etc/sudoers.d/wheel
 
 # Create needed folders
-RUN mkdir /builder /builder/plugins /builder/build /builder/distfiles /builder/cache /builder/repository /builder/sources
+RUN mkdir /builder /builder/plugins /builder/build /builder/build/qubeized_images /builder/distfiles /builder/cache /builder/repository /builder/sources
```

Comment:
Where is this needed?

Reply:
When building a template using docker executor, otherwise you get:
```
10:23:54,360 [executor:docker:bf3ebebfda] No such container:path: bf3ebebfdaf24d4c49ae999c85e3d56fb5215cb985b1b5efb99933f3adf612d0:/builder/build/qubeized_images
Error: fedora-37: Failed to build template.
Traceback (most recent call last):
  File "/home/user/qubes-builderv2/qubesbuilder/executors/container.py", line 104, in copy_in
    subprocess.run(cmd, check=True, capture_output=True)
  File "/usr/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['docker', 'cp', '/home/rene/git/qubes-builderv2/artifacts/templates/qubeized_images/fedora-37/root.img', 'bf3ebebfdaf24d4c49ae999c85e3d56fb5215cb985b1b5efb99933f3adf612d0:/builder/build/qubeized_images/fedora-37']' returned non-zero exit status 1.
```